Stable-Branch-Strategy.md

```diff
@@ -0,0 +1,103 @@
+
+# Overview
+
+As detailed in the [release documentation](https://github.com/kata-containers/documentation/blob/master/Releases.md),
+the Kata Containers project makes use of semantic versioning.  A release version is described
+by MAJOR.MINOR.PATCH.  Early in the project, we will introduce many new features which will require
+minor version changes as well as bug fixes.  To facilitate a stable user environment, Kata will
+begin to provide stable branch based releases as well as a master branch release.
+
+## Stable branch patch criteria
+
+No new features should be introduced to stable branches.  The intention is to limit risk to users,
+providing only bug and security fixes.
+
+## Branch Management
+Kata Containers will maintain two stable release branches in addition to the master branch.
+Once a new MAJOR or MINOR release is created from master, a new stable branch will be created for
+the prior MAJOR or MINOR release and the older stable branch will no longer be maintained.
+
+A couple of examples are provided to help clarify this process.
+
+### New bug fix introduced
+
+A bug fix is submitted against the runtime which does not introduce new inter-component dependencies.
+This fix will be applied to both the master and stable branches, and there is no need to create a new
+stable branch.
+
+| Branch | Original version | New version |
+|--|--|--|
+| `master` | `1.3.0-rc0` | `1.3.0-rc1` |
+| `stable-1.2` | `1.2.0` | `1.2.1` |
+| `stable-1.1` | `1.1.2` | `1.1.3` |
+
+
+### New release made feature or change which  adds new inter-component dependency
+
+A new feature is introduced which adds a new inter-component dependency. In this case a new stable
+branch is created (stable-1.3) starting from master and the older stable branch (stable-1.1)
+is dropped from maintenance.
+
+
+| Branch | Original version | New version |
+|--|--|--|
+| `master` | `1.3.0-rc1` | `1.3.0` |
+| `stable-1.3` | N/A| `1.3.0` |
+| `stable-1.2` | `1.2.1` | `1.2.2` |
+| `stable-1.1` | `1.1.3` | (unmaintained) |
+
+Note, the stable-1.1 branch will still exist with tag 1.1.3, but under current plans it would
+not be maintained further.
+   
+## Backporting Process 
+
+Development will occur against the master branch and applicable code commits should also be submitted
+against the stable branches. Some guidelines:
+ (1) Only bug and security fixes which do not introduce inter-component dependencies are
+ candidates for stable branches. These PRs should be marked with "bug" in GitHub.
+ (2) Once a PR is created against master which meets requirement of (1), a comparable one
+ should also be submitted against the stable branches. It is the responsibility of the submitter
+ to apply their pull request against stable, and it is the responsibility of the
+ reviewers to help identify stable-candidate pull requests.
+ 
+## Continuous Integration Testing
+
+The test repository will also be forked to create stable branches from master. Full CI
```

Comment:
Lines 66-67 suggested rewrite: The test repository is forked to create stable branches from master. Full CI runs on each stable and master PR using its respective tests repository branch.

Reply:
I'm not sure if we have all agreed to this yet, but I think we should also fork the documentation repo: consider how useful that would be not only for release-specific installation guides, but also:

- release-specific limitations docs

  kinda essential ;)

- release-specific developer guide

  very useful as we don't want to have to say things in there like,

  > if you are running a release >= 1.2.3 and < 9.2.6, run these commands...
  > However, if you are running release 3.99.6, run the following commands instead.

  with a fork tied to the release in question, that problem goes away.